src/server/views/transition-page.ejs

```diff
@@ -31,7 +31,7 @@
         </div>
     </div>
     <div class="bottom-half">
-        <p>You will be redirected in <span id="countdown-seconds"></span> second<span id="s">s</span></p>
+        <p id="url" data-href="<%- escapedLongUrl %>">You will be redirected in <span id="countdown-seconds">8</span> second<span id="s">s</span></p>
```

Comment:
setting `data-href` in an arbitrary html element was the best hack I could think of to pass data into gtag.ejs. I borrowed this idea from @LoneRifle, but all suggestions for improvements are welcome